protos/sources/blockexplorer/api/v1/blockexplorer.proto

```diff
@@ -59,12 +60,10 @@ message GetTransactionResponse {
 }
 
 message ListTransactionsRequest {
-  // Number of transactions to be returned from the blockchain.
-  // This is deprecated, use first and last instead.
-  uint32 limit = 1 [deprecated = true];
-  // Optional cursor to paginate the request
+  reserved 1;
+  // Optional cursor to paginate the request. It can be used in conjunction with `after` cursor.
```

Comment:
```suggestion
  // Cursor to paginate the request. It can be used in conjunction with the `after` cursor.
```
If it's marked as optional, it doesn't need to be in the doc string.